### DISCUSSION TITLE
Support testing on x86 linux

### DISCUSSION THREAD

**Discussion by iliya-malecki** [PyO3/maturin#1934] [Ideas]
Hi! Im wondering, is it a good idea to add a little something to the generated CI that would test stuff on x86 linux?
I mean, i dont know much about the topic of actual emulation vs using a 32 bit system but i mean, the pointer size is gonna be 32 bit right? heres a setup im using in github actions:
```yaml
- name: pytest
        if: ${{ matrix.target == 'x86' }}
        run: >
          docker run
          --rm
          --workdir /github/workspace
          -v "/var/run/docker.sock":"/var/run/docker.sock"
          -v "/home/runner/work/peroxymanova/peroxymanova":"/github/workspace"
          i386/python:latest
          sh -c
          "
            ls &&
            find . -maxdepth 2 -type d -ls &&
            pip install peroxymanova --find-links dist --force-reinstall --verbose --no-index --no-deps &&
            pip install peroxymanova &&
            pip install -r pytests/requirements.txt &&
            pip install pytest &&
            pytest
          "
```

**Comment by messense** [comment]
There is not much point in investing into 32-bit Linux now that `manylinux_2_28` has dropped its support: https://github.com/pypa/manylinux?tab=readme-ov-file#manylinux_2_28-almalinux-8-based.

And CentOS 7 will be [EOL soon](https://www.redhat.com/en/topics/linux/centos-linux-eol) which will make targeting `manylinux2014` harder in the near future.

**Comment by iliya-malecki** [comment]
Should I ever worry about testing on any 32 bit system? There's an oldschool mainframe in the testing matrix after all

**Reply by messense** [reply]
I'd say don't worry about it unless there are real world problems with it.
